style.css

```diff
@@ -0,0 +1,169 @@
+@media only screen and (min-width: 768px) {
+  
+  
+}
+@media only screen and (min-width: 375px) {
+  
+  
+}
+body{
+min-height: 100vh;
+}
```

Comment:
- [ ] Kindly use the correct font family for all elements by updating one selector here. You should omit repeating yourself for best practices. You can use google fonts and search for the Poppins family and then, follow the instructions there to import it to your project.
your font: 
![image](https://user-images.githubusercontent.com/91486791/219351706-2ead304d-2c86-4154-92db-5b203d6d78d7.png)
figma font: 
![image](https://user-images.githubusercontent.com/91486791/219351780-edd287b8-dc49-49a9-a952-aa4d47f7c56d.png)